http-servers/echo-server.js

```diff
@@ -0,0 +1,7 @@
+const http = require('http');
+const port = process.env.port || 8080;
+
+http.createServer((req, res) => {
+    res.writeHead(200, { 'Content-Type': req.headers["content-type"] || 'text/plain' });
```

Comment:
It is consider a best practice to form headers object outside of writeHead.
it improves readability and provides more control in case of several different headers.